lib/v1.0/python/pprzlink/udp.py

```diff
@@ -0,0 +1,153 @@
+"""
+PPRZLINK UDP to Python interface
+"""
+from __future__ import absolute_import, division, print_function
+
+import threading
+import socket
+
+# used to run the test from this directory
+# any better implementation ?
+import os
+import sys
+PACKAGE_PARENT = '..'
+SCRIPT_DIR = os.path.dirname(os.path.realpath(os.path.join(os.getcwd(), os.path.expanduser(__file__))))
+sys.path.append(os.path.normpath(os.path.join(SCRIPT_DIR, PACKAGE_PARENT)))
```

Comment:
Yeah, it won't work if you don't add .../pprzlink/lib/v1.0/python to you PYTHONPATH before.
But the relative import should work as expected if you do that, e.g. if you want to use the UDP interface in other code..
So for just running the tests, I would say add a README.md to the pprzlink dir that explains that you need to set your PYTHONPATH if you want to run the tests...